src/countryCode.js

```diff
@@ -0,0 +1,1486 @@
+// @flow
+export type CountryCode = {
+  isoCountryCode: string,
+  callingCode: string,
+  flag: string,
+  name: string,
+};
+
+export const countryCode: CountryCode[] = [
+  {
+    isoCountryCode: "AD",
+    callingCode: "376",
+    flag: "🇦🇩",
```

Comment:
Are you sure all devices can render the all emoji flags?

Reply:
After testing, the flag emoji do not fully support android 4.4 and 5.0.
I will ask @frank-oursky  to give me all flag png file.

Reply:
What is our standard on supporting android version @royuen ?

Reply:
normally 4.3

Reply:
Okay, than we will need the png file.